scbd/absch.cbd.int
# chm-396  add chm content to  general library records page

### General description
add chm content to general library records page.
please review all the fields.

### Designs
use same design in previous page ( Virtual Resource Record in search )

### Testing instructions
search --> virtual resource record
click every record, check the chm content.
compare with previous chm website code:  https://github.com/scbd/chm.cbd.int/blob/master/app/directives/formats/views/resource.html
compare with data get from https://chm.cbd.int/api/v2013/documents/:id 



<img width="682" alt="image" src="https://github.com/scbd/absch.cbd.int/assets/65098066/15293ee6-cfd9-47aa-9861-45d85bec2833">
<img width="697" alt="image" src="https://github.com/scbd/absch.cbd.int/assets/65098066/54105c69-77d4-4efd-92e8-a6004a6f0f45">

<img width="704" alt="image" src="https://github.com/scbd/absch.cbd.int/assets/65098066/12faaac4-df26-49d9-adc1-a1aa3d2a2362">
